src/hooks/api/functions/serialize.ts

```
@@ -7,21 +7,29 @@ import {
 
-export async function serialize(client: Client, messages: WAMessage) {
+export async function serialize(client: WASocket, messages: WAMessage) {
   const normalizedMessages = {
     ...messages,
     message: normalizeMessageContent(messages?.message),
   };
   const { key, message, ...msg } = normalizedMessages;
-  const { prefix, mode, sudo } = await getConfig();
-  const owner = numToJid(client?.user!.id);
+  const { prefix, mode } = await getSettings();
+  const owner = parseJid(client?.user!.id);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)